trinity/sync/full/hexary_trie.py

```diff
@@ -99,15 +99,15 @@ def _get_children(node: Hash32, depth: int
                 # this is a reference to another node.
                 references.append((depth + 1, sub_node))
             else:
-                sub_references, sub_leaves = _get_children(sub_node, depth)
+                sub_references, sub_leaves = _get_children(sub_node, depth)  # type: ignore
                 references.extend(sub_references)
-                leaves.extend(sub_leaves)
+                leaves.extend(sub_leaves)  # type: ignore
```

Comment:
Any chance we can get comments for why these are ignored.

Reply:
I think it is just me not fully grokking the implementation of this function. I'll try to put some meaningful comment on it.